src/Citeproc/Types.hs

```diff
@@ -146,18 +148,26 @@ data CiteprocOptions =
   CiteprocOptions
```

Comment:
thinking of adding a `linkPrefixes` option to CiteprocOptions to control whether or not URL prefixes are added to links (e.g. `https://doi.org/` in front of a DOI)

Reply:
Isn't this something that is handled in the style itself?

Reply:
Oh, I get it now, thanks.  You're saying that the CSL itself decides whether the DOI should be prefixed with `https://doi.org/` or `DOI:` or something else.  We should only be changing the link target, not the anchor.  This is probably the reason for some of the failing tests in my most recent commit.